javalib/src/main/scala/java/net/PlainSocketImpl.scala

```diff
@@ -173,7 +177,7 @@ private[net] class PlainSocketImpl extends SocketImpl {
   override def connect(address: SocketAddress, timeout: Int): Unit = {
     val inetAddr = address.asInstanceOf[InetSocketAddress]
     val hints    = stackalloc[addrinfo]
-    val ret      = stackalloc[Ptr[addrinfo]]
+    val ret      = stdlib.malloc(sizeof[Ptr[Byte]]).cast[Ptr[Ptr[addrinfo]]] //stackalloc[Ptr[addrinfo]]
```

Comment:
Stackalloc and malloc are not 1-to-1 equivalents. malloc need to be cleaned up manually otherwise this line introduces a memory leak.